common/focuses/wc_magic_lifestyle_focuses.txt

```diff
@@ -0,0 +1,927 @@
+﻿light_magic_focus_1 = {
```

Comment:
There may be more, I searched for `wc` in the files.
```
[20:35:09][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: elemental_spirit_magic_lifestyle_perks, near line: 5
Unknown trigger type: elemental_spirit_magic_lifestyle_perk_points, near line: 6
Unknown trigger type: elemental_spirit_magic_lifestyle_xp, near line: 7
" in file: " file: common/focuses/wc_magic_lifestyle_focuses.txt line: 534; MAGIC_lifestyle_or_focus_is_valid_trigger" near line: 8
[20:35:09][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: elemental_decay_magic_lifestyle_perks, near line: 5
Unknown trigger type: elemental_decay_magic_lifestyle_perk_points, near line: 6
Unknown trigger type: elemental_decay_magic_lifestyle_xp, near line: 7
" in file: " file: common/focuses/wc_magic_lifestyle_focuses.txt line: 598; MAGIC_lifestyle_or_focus_is_valid_trigger" near line: 8
[20:35:09][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: elemental_spirit_magic_lifestyle_perks, near line: 5
Unknown trigger type: elemental_spirit_magic_lifestyle_perk_points, near line: 6
Unknown trigger type: elemental_spirit_magic_lifestyle_xp, near line: 7
" in file: " file: common/focuses/wc_magic_lifestyle_focuses.txt line: 634; MAGIC_lifestyle_or_focus_is_valid_trigger" near line: 8
[20:35:09][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: elemental_decay_magic_lifestyle_perks, near line: 5
Unknown trigger type: elemental_decay_magic_lifestyle_perk_points, near line: 6
Unknown trigger type: elemental_decay_magic_lifestyle_xp, near line: 7
" in file: " file: common/focuses/wc_magic_lifestyle_focuses.txt line: 698; MAGIC_lifestyle_or_focus_is_valid_trigger" near line: 8
[20:35:09][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: elemental_spirit_magic_lifestyle_perks, near line: 5
Unknown trigger type: elemental_spirit_magic_lifestyle_perk_points, near line: 6
Unknown trigger type: elemental_spirit_magic_lifestyle_xp, near line: 7
" in file: " file: common/focuses/wc_magic_lifestyle_focuses.txt line: 734; MAGIC_lifestyle_or_focus_is_valid_trigger" near line: 8
[20:35:09][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: elemental_decay_magic_lifestyle_perks, near line: 5
Unknown trigger type: elemental_decay_magic_lifestyle_perk_points, near line: 6
Unknown trigger type: elemental_decay_magic_lifestyle_xp, near line: 7
" in file: " file: common/focuses/wc_magic_lifestyle_focuses.txt line: 798; MAGIC_lifestyle_or_focus_is_valid_trigger" near line: 8
[20:35:09][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: elemental_spirit_magic_lifestyle_perks, near line: 5
Unknown trigger type: elemental_spirit_magic_lifestyle_perk_points, near line: 6
Unknown trigger type: elemental_spirit_magic_lifestyle_xp, near line: 7
" in file: " file: common/focuses/wc_magic_lifestyle_focuses.txt line: 834; MAGIC_lifestyle_or_focus_is_valid_trigger" near line: 8
[20:35:09][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: elemental_decay_magic_lifestyle_perks, near line: 5
Unknown trigger type: elemental_decay_magic_lifestyle_perk_points, near line: 6
Unknown trigger type: elemental_decay_magic_lifestyle_xp, near line: 7
" in file: " file: common/focuses/wc_magic_lifestyle_focuses.txt line: 898; MAGIC_lifestyle_or_focus_is_valid_trigger" near line: 8
```

Reply:
Should be fixed now